### PR TITLE
Added ink-select-input-horizontal to useful components

### PR DESCRIPTION
**Pull request by H3RSKO**
Added ink-select-input-horizontal to useful components on Readme

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1727,6 +1727,7 @@ You can even inspect and change the props of components, and see the results imm
 - [ink-text-input](https://github.com/vadimdemedes/ink-text-input) - Text input.
 - [ink-spinner](https://github.com/vadimdemedes/ink-spinner) - Spinner.
 - [ink-select-input](https://github.com/vadimdemedes/ink-select-input) - Select (dropdown) input.
+- [ink-select-input-horizontal](https://github.com/H3RSKO/ink-select-input-horizontal) - Mod of the above componenet adding color and orientation customization.
```
Comment (sindresorhus) [inline readme.md:1730]: "mod" is not a word, and you have a typo.

**Comment by sindresorhus** [comment]
Why not contribute to the original component instead?

File: readme.md (hunk 1 above)
Comment (H3RSKO) [reply readme.md]: Fixed typo and changed 'mod' to modification. Thought I would be going over line length but apparently not. 

Regarding the original component, we had put this together for our own project which had different default settings, and only after added in the customizability. 

I can work on modifying the original instead. Just give me a few days.

**Comment by vadimdemedes** [comment]
@sindresorhus I think we've discussed this in the original repo somewhere and I advised to make it a separate package to keep the original one simple. I think the ideal strategy here is to keep these components separate, but extract their functionality into a React hook and reuse it between the two.

**Comment by vadimdemedes** [comment]
Hey @H3RSKO, there are some issues in your readme:

1. Indentation is off in the `require()` code example
2. "Looking for a version compatible with Ink 2.x?" link points to `ink-text-input`.
3. Could you link to [ink-select-input](https://github.com/vadimdemedes/ink-select-input) in "This is a modified version of the original ink-select-input" text?
4. "Default" is on the same line as "Type" in [initialIndex](https://github.com/H3RSKO/ink-select-input-horizontal#initialindex) description.

**Comment by sindresorhus** [comment]
Doesn't seem like it would add a lot of complexity though. It just a `direction` prop with some extra handling: https://github.com/H3RSKO/ink-select-input-horizontal/commit/27a1f085a7416c22058ffe451af4b31f7d692122

**Comment by vadimdemedes** [comment]
That's true, didn't think it would be so simple. Sorry @H3RSKO, would you be able to add it to `ink-select-input` itself?

**Comment by H3RSKO** [comment]
Hey guys, I posted a PR here: https://github.com/vadimdemedes/ink-select-input/pull/35#issue-498076509

As i mentioned there, I was having trouble with NPM link so please confirm it works before adding. 

Thanks!

**Comment by vadimdemedes** [comment]
Cool, closing this one then, let's continue discussion in that PR.

**Pull request closed without merging**